### PR TITLE
feat(docs): Aws Resource Guide

### PR DESCRIPTION
**Pull request by anjafr**
# Description

Adds a section on how to contribute collecting AWS Resources

# To-Dos

- [x] Format with `yarn format`
- [x] Lint with `yarn lint`
- [x] Build successfully with `yarn build`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://resoto.com/code-of-conduct).

### CODE REVIEW

File: docs/contributing/plugins/index.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+import { useCurrentSidebarCategory } from '@docusaurus/theme-common';
+import DocCardList from '@theme/DocCardList';
+```
 
 # Contributing to Plugins
 
 Collector plugins allow for importing of arbitrary resources into Resoto in graph form. The most common use case is to gather information about cloud accounts and/or resources. However, any data expressible is graph form can be collected—be it social media accounts, software dependency trees, network topology, steps for cooking your favorite food, etc.
 
-Once the graph data is collected and sent to [Resoto Core](../concepts/components/core.md), the power of Resoto's [search syntax](../concepts/search/index.md) is at your fingertips.
+Once the graph data is collected and sent to [Resoto Core](docs/concepts/components/core.md), the power of Resoto's [search syntax](docs/concepts/search/index.md) is at your fingertips.
```
Comment (TheCatLady) [inline docs/contributing/plugins/index.md:12]:
```suggestion
Once the graph data is collected and sent to [Resoto Core](../../concepts/components/core.md), the power of Resoto's [search syntax](../../concepts/search/index.md) is at your fingertips.
```
Markdown links within docs pages need to be relative to the current doc.

File: docs/contributing/plugins/index.md [hunk 2]
```diff
@@ -54,10 +59,12 @@ Please refer to the [example collector plugin](https://github.com/someengineerin
 
 ### Testing Plugins
 
-To test a plugin, simply launch Resoto and trigger the collect action manually by executing `workflows run collect` in the [shell](../concepts/components/shell.md). Once the collection is complete, you execute `search (<plugin_resource_type>)` to see the newly collected resources.
+To test a plugin, simply launch Resoto and trigger the collect action manually by executing `workflows run collect` in the [shell](docs/concepts/components/shell.md). Once the collection is complete, you execute `search (<plugin_resource_type>)` to see the newly collected resources.
```
Comment (TheCatLady) [inline docs/contributing/plugins/index.md:62]:
```suggestion
To test a plugin, simply launch Resoto and trigger the collect action manually by executing `workflows run collect` in the [shell](../../concepts/components/shell.md). Once the collection is complete, you execute `search (<plugin_resource_type>)` to see the newly collected resources.
```

File: docs/contributing/plugins/index.md [hunk 3]
```diff
@@ -1,20 +1,25 @@
----
-sidebar_label: Plugins
----
+# Plugins
+
+```mdx-code-block
+import { useCurrentSidebarCategory } from '@docusaurus/theme-common';
```
Comment (TheCatLady) [inline docs/contributing/plugins/index.md:4]:
```suggestion
```

File: docs/contributing/plugins/index.md [hunk 4]
```diff
@@ -54,10 +59,12 @@ Please refer to the [example collector plugin](https://github.com/someengineerin
...
+To test a plugin, simply launch Resoto and trigger the collect action manually by executing `workflows run collect` in the [shell](docs/concepts/components/shell.md). Once the collection is complete, you execute `search (<plugin_resource_type>)` to see the newly collected resources.
 
 ### Tips
 
 - **Resource properties types must be globally unique.** For example, if there is a property `status: str` defined by some other plugin and you add a property `status: int`, the collection will fail. The reason for such behavior is that all resource properties are indexed for search.
 - **Try to not introduce unique property names.** Only do it in event that types are not compatible with existing properties. For example, use `status: str` instead of `plugin_name_resource_name_status: int`.
 - **Don't forget to call `sanitize(graph)` before checking its properties in tests.** Please refer to the [DigitalOcean plugin](https://github.com/someengineering/resoto/tree/main/plugins/digitalocean) as an example.
+
+<DocCardList items={useCurrentSidebarCategory().items}/>
```
Comment (TheCatLady) [inline docs/contributing/plugins/index.md:70]:
```suggestion
<DocCardList />
```

**Review by TheCatLady — CHANGES_REQUESTED** [review]
I will also push some edits to your new doc, but here are some comments FYI

File: docs/contributing/plugins/index.md [hunk 5]
```diff
@@ -1,20 +1,25 @@
----
-sidebar_label: Plugins
----
+# Plugins
```
Comment (TheCatLady) [inline docs/contributing/plugins/index.md:1]: Need to remove this heading, restore the above frontmatter, and move the import below the "Contributing to Plugins" heading below.

**Comment by anjafr** [comment]
@TheCatLady 
> I will also push some edits to your new doc, but here are some comments FYI

Please do! I have no idea what I'm doing 😆

**Comment by TheCatLady** [comment]
> Please do! I have no idea what I'm doing 😆

No worries, I can take over from here :)

**Review by aquamatthias — APPROVED** [review]

**Comment by aquamatthias** [comment]
Thanks! Merged the page to get the link. Please create a separate PR for any outstanding items.

**Pull request merged**